pyphare/pyphare/pharesee/hierarchy.py

```diff
@@ -149,9 +149,29 @@ def __init__(self, layout, field_name, data, **kwargs):
 
         self.dataset = data
 
+    def compare(self, that, atol=1e-16, rtol=0):
+        if isinstance(that, FieldData):
+            assert self.name == that.name
+            # drop nans
+            a = self.dataset
+            b = that.dataset
+            mask = ~(np.isnan(a) | np.isnan(b))
```

Comment:
this can probably be dropped after https://github.com/PHAREHUB/PHARE/pull/697 is merged
TBC

Reply:
#697 is merged, any idea?

Reply:
probably droppable so, to test